nvflare/fuel/f3/drivers/connector.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from dataclasses import dataclass
+from enum import Enum
+
+from nvflare.fuel.f3.drivers.net_utils import short_url
+
+
+class Mode(Enum):
+    ACTIVE = 0
+    PASSIVE = 1
+
+
+@dataclass
+class Connector:
+    """Connector information"""
+
+    handle: str
+    # noinspection PyUnresolvedReferences
+    driver: "Driver"
+    params: dict
+    mode: Mode
+    total_conns: int
+    curr_conns: int
```

Comment:
why do we need to have total_conns VS curr_conns?

Reply:
Total_conns is how many connection created (including disconnected ones), curr_conns is the live connected ones. Total_conns can be used to judge network health.